src/Umbraco.Web.UI.NetCore/Startup.cs

```diff
@@ -47,7 +48,10 @@ public void ConfigureServices(IServiceCollection services)
             services.AddUmbracoCore(_env, out var factory);
             services.AddUmbracoWebsite();
 
-            services.AddMvc();
+            services.AddMvc(options =>
```

Comment:
I think this should be removed.. I think we can explicit add this modelbinder to the required actions, like we do for `ContentItemBinder`

Reply:
Have removed.